### PR TITLE
Create e-c-centric.md

### PR DESCRIPTION
**Pull request by e-c-centric**
## Checklist before submitting proposal:

- [#] I am a current high school student (or younger).
- [#] I have submitted an RSVP at <https://hackclub.com/winter>
- [#] I am a member of the Hack Club Slack (Join at <https://hackclub.com/slack>)
- [#] I have self-reviewed my project proposal.

### CODE REVIEW

**Review by bellesea — CHANGES_REQUESTED** [review]
hiii @e-c-centric thank you sm for sharing this project idea, it's super cool! i'm wondering, are you referencing any guides/existing projects for this build?

also, i saw that within your bundles you'd be purchasing an Arduino UNO and an Arduino Nano RP2040 Connect, which do you plan on using?

thanks :)

**Comment by e-c-centric** [comment]
Hiii @bellesea , thanks for your feedback :)

I am not referencing any guide per se, but I am referencing a lot of projects on the Arduino Project Hub. Also, I have older friends in uni who have worked/are working on IoT projects. I've got a lot of information from them on how to proceed. 

I plan on using the Arduino Nano RP2040 Connect because of its processor and capabilities. I also own an ESP32 that I'll try this project with. I'll likely go with the Arduino Nano because it significantly reduces the hassle through its Arduino Cloud connectivity and the power of the Raspberry Pi processor. the Arduino Uno comes together with a bundle of other stuff I need, so I can't exactly remove it. In the course of building this project, I might find a use for it 😃, maybe for a control experiment

thankss

**Comment by bellesea** [comment]
> Hiii @bellesea , thanks for your feedback :)
> 
> I am not referencing any guide per se, but I am referencing a lot of projects on the Arduino Project Hub. Also, I have older friends in uni who have worked/are working on IoT projects. I've got a lot of information from them on how to proceed.
> 
> I plan on using the Arduino Nano RP2040 Connect because of its processor and capabilities. I also own an ESP32 that I'll try this project with. I'll likely go with the Arduino Nano because it significantly reduces the hassle through its Arduino Cloud connectivity and the power of the Raspberry Pi processor. the Arduino Uno comes together with a bundle of other stuff I need, so I can't exactly remove it. In the course of building this project, I might find a use for it 😃, maybe for a control experiment
> 
> thankss

hey @e-c-centric ahh that makes sense :) Unfortunately, i don't think we can approve a budget to buy two arduinos when you only need one?

what do you think about getting the IOT bundle https://store.arduino.cc/products/iot-bundle?variant=42614286844055 then purchasing other specific sensors that you're looking for? 

thank you :)

**Comment by e-c-centric** [comment]
@bellesea thanks. I'll rework my budget by close of day today.

**Comment by bellesea** [comment]
> @bellesea thanks. I'll rework my budget by close of day today.

great, no rush :) thanks sm!

**Comment by bellesea** [comment]
heyy just checking, do you have what you need to program the arduino? (eg. a usb port on your computer)

**Comment by cfanoulis** [comment]
Hello Elikem ! It has been a while since we've heard from you - and we need to know if you're still interested in a winter grant. If you are - please reply to this PR (or this email) within the next 48 hours - else we will be closing this proposal. Thanks :)

**Comment by e-c-centric** [comment]
Hiiii @cfanoulis @bellesea , really sorry for my absence. I was really ill over the week, and just getting better.

**Comment by e-c-centric** [comment]
> heyy just checking, do you have what you need to program the arduino? (eg. a usb port on your computer)

yes, I do

**Comment by cfanoulis** [comment]
hey @e-c-centric, no worries! just needed to know if you're still interested in making this :) belle will get back to you on monday, as they are off for the weekend

have a hacky, uh, recovery? lol

**Comment by bellesea** [comment]
hey @e-c-centric oh no :( hope you feel better soon! when do you think you'd be able to update the budget based on previous comments?

**Comment by e-c-centric** [comment]
> hey @e-c-centric oh no :( hope you feel better soon! when do you think you'd be able to update the budget based on previous comments?

I'll do that by end of day today. Thanks

**Comment by e-c-centric** [comment]
done :)

**Review by bellesea — CHANGES_REQUESTED** [review]
hey @e-c-centric thanks sm for pushing the changes!

a few more questions:
- are you referencing any guides of sorts?
- why do you need the screwdriver set haha?

thank you sm!

**Comment by e-c-centric** [comment]
> hey @e-c-centric thanks sm for pushing the changes!
> 
> a few more questions:
> 
> * are you referencing any guides of sorts?
> * why do you need the screwdriver set haha?
> 
> thank you sm!

Hiii @bellesea , on the screwdriver set, I was planning to build a housing for the different parts of my project (microcontroller and sensors) because a lot of the data will be collected outside, and unmonitored. I was going to use wood, plastic/plexiglass, maybe some metal and glass. My school's workshop gives me access to these materials, and welding and soldering stations. What we don't have however, are screwdrivers (because students keep taking them away 😂). I foresee I'll have to join stuff a fair bit to make the housing, and I don't want to use nails or any other permanent joint because I might need to take it apart to reach the components.

On a reference guide, I'm referencing several IOT projects on the Arduino hub, as well as projects students I know in uni have undertaken.

File: e-c-centric.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+name: "@e-c-centric"
+project: "IOT Weather Monitoring and Prediction Station"
+---
+
+# Project Name
```
Comment (bellesea) [inline e-c-centric.md:6]:
```suggestion
# IOT Weather Monitoring and Prediction Station
```

**Review by bellesea — APPROVED** [review]
@e-c-centric haha i know what you mean about the screwdrivers!

an awesome member of @hackclub/bank-ops will be reaching out soon with next steps :)

approving to move forward but can you please:
- add a total to your budget
- add your plan for casing under your plan?

thank you sm!

**Comment by e-c-centric** [comment]
hiii @bellesea thank you so much :) . I've made the changes requested. I'm not sure if I committed them right though.

>

**Comment by bellesea** [comment]
@e-c-centric this looks good, thanks!

**Review by MelSmith104 — COMMENTED** [review]
hey @e-c-centric! this project sounds so useful! i rely on my inaccurate (at times) weather app but this seems like a much better method :) im dm'ing you right now on slack to chat about next steps!

**Comment by MelSmith104** [comment]
hey @e-c-centric could you please let me know your slack id :)

**Comment by e-c-centric** [comment]
> hey @e-c-centric could you please let me know your slack id :)

hii @MelSmith104 it's U04H6MP70RK

**Comment by MelSmith104** [comment]
just dm'ed you!

**Comment by MelSmith104** [comment]
closing as we can only move forward with projects made by teens that are still in high school. really appreciate the time you put into your proposal and hope you decide to build it down the road!

**Pull request closed without merging**